runtime/integration-test/src/proxy.rs

```diff
@@ -5,73 +5,6 @@ type SystemError = frame_system::Error<Runtime>;
 const ASSET_ID_1: u32 = 1;
 const LP_ASSET_ID: u32 = 2;
 
-#[test]
```

Comment:
shouldnt we rather align those ?

Reply:
we already have the same cases covered by the other test:
add proxy
proxy OK, inner call executed when extrinsic is allowed for proxy type
proxy OK, inner call filtered out when extrinsic is not allowed for proxy type
remove proxy
proxy FAIL with no proxied account